client/src/components/CMenu.style.js

```diff
@@ -0,0 +1,95 @@
+import styled from 'styled-components';
+
+export const MenuContainer = styled.div`
```

Comment:
this can probably be a ```main``` tag

Reply:
no it's just for the menuburger

Reply:
oops! i guess it would be best put in a nav tag then :)

🐙